Dockerfile

```diff
@@ -9,17 +16,17 @@ WORKDIR /workdir
 # currently its just 'python3', 'python3-pip', 'sqlite3', and 'ffmpeg'
 # if more programs are needed they should be added here
 RUN apt update
-RUN DEBIAN_FRONTEND=noninteractive apt install -y --no-install-recommends python3 python3-pip sqlite3 wget && apt clean
-RUN DEBIAN_FRONTEND=noninteractive apt install -y ffmpeg libx264-155 libx265-165 && apt clean
```

Comment:
libx264 is necessary for the video hosting, it does not come with ffmpeg

Reply:
We’ll have to figure out a stable way to include it.  I removed libx264-155 and libx265-165 because they don’t work anymore.  It seems like they don’t keep older versions available.

Reply:
Does a `*` glob work with `apt install`? Or we can just switch to alpine or arch.

Reply:
It looks to me that installing libx265-dev and libx264-dev will install the corresponding shared libraries.  That's probably the simplest way to handle this, although it comes with a little bloat.  The * does work but installs some other stuff as well (including the -dev packages).